### PR TITLE
Add Kotlin code for the array.md

### PR DESCRIPTION
**Pull request by nanshaws**
在创建数组章节添加了kotlin代码

### CODE REVIEW

**Comment by krahets** [comment]
@curtishd Please review this PR at your convenience, thanks!

**Comment by curtishd** [comment]
Confirm.

**Comment by krahets** [comment]
> Confirm.

You can "approve" this PR at https://github.com/krahets/hello-algo/pull/1156/files#submit-review

**Comment by krahets** [comment]
This PR will be merged after adding Kotlin code blocks to the documents.

**Review by curtishd — APPROVED** [review]

File: docs/chapter_array_and_linkedlist/array.md [hunk 1]
```diff
@@ -105,6 +105,25 @@
     int nums[5] = { 1, 3, 2, 5, 4 };
     ```
 
+=== "Kotlin"
+
+    ```kotlin title="array.kt"
+    // 初始化数组
+    val arr = arrayOf(1, 2, 3, 4, 5)
```
Comment (krahets) [inline docs/chapter_array_and_linkedlist/array.md:112]: The Kotlin code is not consistent with other languages. Please fix it referring to Python or Java code.

**Comment by krahets** [comment]
@nanshaws Please merge the main branch of the origin repo to yours to resolve the conflicts.

**Pull request closed without merging**